bundle/edu.gemini.itc/src/main/scala/edu/gemini/itc/operation/Slit.scala

```diff
@@ -26,16 +26,17 @@ sealed trait Slit {
 
 }
 
-/** Slit that covers exactly one pixel². This is used to calculate the peak pixel flux for a single pixel. */
-final case class OnePixelSlit(pixelSize: Double) extends Slit {
+/** Slit that is 1-pixel long for calculating the peak pixel flux. */
+final case class OnePixelSlit(width: Double, pixelSize: Double) extends Slit {
 
-  val width                   = pixelSize
-  val length                  = pixelSize
-  val area                    = width * length
+  // width = slit width in arcseconds
+  // pixelSize = pixel size in arcseconds
+  val length                  = pixelSize        // slit length in arcseconds
```

Comment:
This is not necessary for this PR but we may want to have these in squants-like quantities. It would make some operations safer